shanoir-ng-datasets/src/main/java/org/shanoir/ng/examination/service/ExaminationServiceImpl.java

```diff
@@ -121,6 +133,21 @@ public void deleteFromRabbit(Examination exam) throws EntityNotFoundException {
 			return examinationRepository.findByPreclinicalAndStudyIdIn(preclinical, studyIds, pageable);
 		}
 	}
+	
+	@Override
+	public Page<Examination> findPage(final Pageable pageable, String patientName) {
+		if (KeycloakUtil.getTokenRoles().contains("ROLE_ADMIN")) {
+			if (StringUtils.isNotEmpty(patientName) && patientName.length() <= 64) {
```

Comment:
Why this 64 char limit ?

Reply:
64 is the max length for DICOM patient name, see DICOM spec